Libs/GModule-2.0/GModule-2.0.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
GModule is not used anywhere. Either delete it completely or add to projects which should be generated.